### PR TITLE
[IMP] hr: remove HR Department created and adding Department created.

### PR DESCRIPTION
**Pull request by mag-odoo**
Task:https://www.odoo.com/web#id=21114&view_type=form&model=project.task&action=327&menu_id=
Pad:https://pad.odoo.com/p/r.2wUdTYOrEzDvJNYq

### CODE REVIEW

File: addons/hr/hr.py [hunk 1]
```diff
@@ -272,7 +272,7 @@ def _message_get_auto_subscribe_fields(self, cr, uid, updated_fields, auto_follo
 
 class hr_department(osv.osv):
     _name = "hr.department"
-    _description = "HR Department"
+    _description = "Department"
```
Comment (atp-odoo) [inline addons/hr/hr.py]: i don't think this one is proper solution..
there is already  create method in hr department.
you should just pass context before super call of create and then call message_post like this way : 
        create_context = dict(context, **mail_create_nolog=True**)
        new_id = super(hr_department, self).create(cr, uid, vals, context=**create_context**)
        self.message_post(cr, uid, new_id, **body=_('Department created')**)

Comment (mart-e) [inline addons/hr/hr.py]: @atp-odoo this is hacking the create to avoid sending a message and then sending it yourself. Why do you want to do that?

Comment (atp-odoo) [inline addons/hr/hr.py]: @mart-e Yes you are right. 
but I thought if  in some-other module 'hr.department' is inherited and override _description attribute with different string then message changed.
-For hack-  there is already create method in hr.department.so I just suggest one more option 
to pass mail_create_nolog=True In context to use custom message and use message_post.

**Comment by dmo-odoo** [comment]
Merged yti's fix at e6c35bc4de034fe420c8b6d3d602464210d52201.

**Pull request closed without merging**